src/components/molecules/UserProfilePicture/UserProfilePicture.tsx

```diff
@@ -129,7 +129,7 @@ export const UserProfilePicture: React.FC<UserProfilePictureProp> = ({
       {userId && (
         <UserReactions
           userId={userId}
-          isMuted={isAudioEffectDisabled}
+          isMuted={isAudioEffectEnabled}
```

Comment:
`isAudioEffectEnabled = true => isMuted = false`
```suggestion
          isMuted={!isAudioEffectEnabled}
```

Reply:
changed names for variables and props in deb69c9 to prevent any negations and make things more clear
tested with music venues/auditorium to confirm that everything is working properly